modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -238,6 +238,14 @@ public static void Print(params object[] what)
             godot_icall_GD_print(GetPrintParams(what));
         }
 
+        /// <summary>
+        /// Prints a string including BBCode
+        /// </summary>
```

Comment:
We try to keep the C# documentation in sync with the GDScript documentation:

```suggestion
        /// <summary>
        /// Converts one or more arguments of any type to string in the best way possible
        /// and prints them to the console. Using BBCode for colors, italics and bold is possible.
        ///
        /// Note: Consider using <see cref="PushError(string)"/> and <see cref="PushWarning(string)"/>
        /// to print error and warning messages instead of <see cref="Print(object[])"/>.
        /// This distinguishes them from print messages used for debugging purposes,
        /// while also displaying a stack trace when an error or warning is printed.
        /// </summary>
        /// <example>
        /// <code>
        /// var a = new int[] { 1, 2, 3 };
        /// GD.PrintRich("a", "b", a); // Prints ab[1, 2, 3]
        /// </code>
        /// </example>
        /// <param name="what">Arguments that will be printed.</param>
```

I noticed that the documentation you added for GDScript uses `print` instead of `print_rich` in the example, so if you change it make sure to update it here too.

Reply:
Thank you for this! Wasn't really sure how to do the documentation for the c# part. I changed the example to the same one being used in @GlobalScope.

Reply:
You still left code from the previous example, and forgot the quotes for the string argument.

Reply:
I'm so sorry about that, I fixed it and added the quotes both here and in the GlobalScope.
Thank you for the help and sorry again for missing this!

Reply:
It's fine, don't worry about it.